addons/account_peppol/models/res_config_settings.py

```diff
@@ -310,8 +293,10 @@ def button_deregister_peppol_participant(self):
 
         # fetch all documents and message statuses before unlinking the edi user
         # so that the invoices are acknowledged
-        self.env['account_edi_proxy_client.user']._cron_peppol_get_message_status()
-        self.env['account_edi_proxy_client.user']._cron_peppol_get_new_documents()
+        self.env.ref('account_peppol.ir_cron_peppol_get_message_status')._trigger()
+        self.env.ref('account_peppol.ir_cron_peppol_get_new_documents')._trigger()
```

Comment:
same comment here about calling the CRONs

Reply:
these are already called for every active use in the database. Although with aboo's changes we will be able to add company domain in master